lib/eos/ecc/point.js

```diff
@@ -1,10 +1,24 @@
-var assert = require('assert')
-var BigInteger = require('bigi/lib/bigi')
+import bigInt from 'big-integer';
+import { assert } from './../helper';
 
-var THREE = BigInteger.valueOf(3)
+const ONE = bigInt.one;
+const ZERO = bigInt.zero;
+const THREE = bigInt(3);
 
-function Point (curve, x, y, z) {
-  assert.notStrictEqual(z, undefined, 'Missing Z coordinate')
+function testBit(bigIntVal, index) {
+  return bigIntVal.toString(2)[index] === '1';
+}
+
+function bufferToBigInt(buffer) {
+  return bigInt(buffer.toString('hex'), 16);
+}
+
+function bigIntToBuffer(bigIntVal, size) {
+  return Buffer.from(bigIntVal.toString(16).padStart(size, '0'), 'hex');
```

Comment:
Likewise - does it not have a function that returns a buffer?

Reply:
No, it doesn't